### PR TITLE
Fix the translations

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Add missing translations

### CODE REVIEW

**Review by andrewleith — CHANGES_REQUESTED** [review]
Still missing one message:
image

**Comment by andrewleith** [comment]
I fixed one string that didn't quite match the translation, as well as the some syntax that was preventing `pybabel` from detecting the missing translations.  Instead of :
`_('translation {'}.format(abc))` it should be `_('translation {'}').format(abc)`

@amazingphilippe , are there translations for these strings?  I couldn't find them in figma!

- `For more information, visit the <a href={}>usage report for {}</a>.`
- `<strong>{}</strong> cannot send any more {} until <strong>April 1, {}</strong>`
- `To send some of these messages now, edit the spreadsheet to <strong>{}</strong> recipients maximum. `

**Review by amazingphilippe — CHANGES_REQUESTED** [review]
We were missing translations in Figma as well:

```csv
"{} cannot send any more {} until April 1, {}","{} ne peut plus envoyer de courriels d’ici le 1<sup>er</sup> avril {}."
"For more information, visit the <a href={}>usage reports</a>.","Pour plus de détails, <a href={}>consulter les rapports d’utilisation</a>."
```
A few notes:
- removed `<strong>`.
- added a `<sup>` for properly formatting "1er" in FR
- removed link text customization with the service name. Less tricky in FR. The service name is also in the sentence before. 

Noticed 2 more things:
- We nest p tags (we shouldn't)
- We use the same font size for both paragraphs. Adding a class `text-title` to the first sentence in each condition would fix that.

File: app/templates/partials/check/too-many-messages-annual.html [hunk 1]
```diff
@@ -6,19 +6,19 @@
   {% set units = _('text messages') %}
 {% endif %}
 
-<p data-testid="exceeds-annual">
+<div data-testid="exceeds-annual" class="text-title">
```
Comment (amazingphilippe) [inline app/templates/partials/check/too-many-messages-annual.html:9]:
```suggestion
<div data-testid="exceeds-annual">
```

File: app/templates/partials/check/too-many-messages-annual.html [hunk 2]
```diff
@@ -6,19 +6,19 @@
...
   {%- if current_service.trial_mode %}
     {{ _("Your service is in trial mode. To send more messages, <a href='{}'>request to go live</a>").format(url_for('main.request_to_go_live', service_id=current_service.id)) }}
   {% else %}
     {% if recipients_remaining_messages > 0 %}
-      <p>{{ _('<strong>{}</strong> can only send <strong>{}</strong> more {} until annual limit resets'.format(current_service.name, recipients_remaining_messages, units)) }}</p>
+      <p>{{ _('<strong>{}</strong> can only send <strong>{}</strong> more {} until annual limit resets').format(current_service.name, recipients_remaining_messages, units) }}</p>
```
Comment (amazingphilippe) [inline app/templates/partials/check/too-many-messages-annual.html:14]:
```suggestion
      <p class="text-title">{{ _('<strong>{}</strong> can only send <strong>{}</strong> more {} until annual limit resets').format(current_service.name, recipients_remaining_messages, units) }}</p>
```

File: app/templates/partials/check/too-many-messages-annual.html [hunk 3]
```diff
@@ -6,19 +6,19 @@
...
       <p>
-        {{ _('To send some of these messages now, edit the spreadsheet to <strong>{}</strong> recipients maximum. '.format(recipients_remaining_messages)) }}
-        {{ _('To send to recipients you removed, wait until <strong>April 1, {}</strong> or contact them some other way.'.format(now().year)) }}
+        {{ _('To send some of these messages now, edit the spreadsheet to <strong>{}</strong> recipients maximum. ').format(recipients_remaining_messages) }}
+        {{ _('To send to recipients you removed, wait until <strong>April 1, {}</strong> or contact them some other way.').format(now().year) }}
       </p>
     {% else %}
-      <p>{{ _('<strong>{}</strong> cannot send any more {} until <strong>April 1, {}</strong>'.format(current_service.name, units, now().year)) }}</p>
-      <p>{{ _('For more information, visit the <a href={}>usage report for {}</a>.'.format(url_for('.monthly', service_id=current_service.id), current_service.name)) }}</p>
+      <p>{{ _('<strong>{}</strong> cannot send any more {} until <strong>April 1, {}</strong>').format(current_service.name, units, now().year) }}</p>
```
Comment (amazingphilippe) [inline app/templates/partials/check/too-many-messages-annual.html:20]:
```suggestion
      <p class="text-title">{{ _('{} cannot send any more {} until April 1, {}').format(current_service.name, units, now().year) }}</p>
```

File: app/translations/csv/fr.csv [hunk 4]
```diff
@@ -2029,6 +2030,11 @@
...
+"These messages exceed the annual limit","Votre limite annuelle ne vous permet pas d’envoyer ces messages."
+"To send some of these messages now, edit the spreadsheet to <strong>{}</strong> recipients maximum. ","Pour envoyer certains de ces courriels maintenant, modifiez la feuille de calcul pour n’inclure que <strong>{}</strong> destinataires au maximum."
+"<strong>{}</strong> can only send <strong>{}</strong> more {} until annual limit resets","<strong>{}</strong> ne peut envoyer que <strong>{}</strong> {} d'ici la réinitialisation de la limite annuelle"
+"To send to recipients you removed, wait until <strong>April 1, {}</strong> or contact them some other way.","Pour envoyer des messages aux destinataires, attendez <strong> le 1er avril {}</strong> ou optez pour une autre méthode de contact."
+"For more information, visit the <a href={}>usage report for {}</a>.","FR: For more information, visit the <a href={}>usage report for {}</a>."
+"<strong>{}</strong> cannot send any more {} until <strong>April 1, {}</strong>","FR: <strong>{}</strong> cannot send any more {} until <strong>April 1, {}</strong>"
```
Comment (amazingphilippe) [inline app/translations/csv/fr.csv:2040]:
```suggestion
"For more information, visit <a href={}>usage reports</a>.","Pour plus de détails, <a href={}>consulter les rapports d’utilisation</a>."
"{} cannot send any more {} until April 1, {}","{} ne peut plus envoyer de {} d’ici le 1<sup>er</sup> avril {}."
```

File: app/templates/partials/check/too-many-messages-annual.html [hunk 5]
```diff
@@ -6,19 +6,19 @@
...
+      <p>{{ _('For more information, visit the <a href={}>usage report for {}</a>.').format(url_for('.monthly', service_id=current_service.id), current_service.name) }}</p>
```
Comment (amazingphilippe) [inline app/templates/partials/check/too-many-messages-annual.html:21]:
```suggestion
      <p>{{ _('For more information, visit <a href={}>usage reports</a>.').format(url_for('.monthly', service_id=current_service.id)) }}</p>
```

**Review by amazingphilippe — CHANGES_REQUESTED** [review]
Suggested some changes. Though I haven't run these and they seem like a lot to suggest in a review...

File: app/translations/csv/fr.csv (hunk 4 above)
Comment (andrewleith) [reply app/translations/csv/fr.csv:2040]: No prob..
FYI, I added the strong tags originally to highlight the most important infomation, because these error messages are so long they seemed hard to parse.

**Review by whabanks — APPROVED** [review]
Tested locally, LGTM!

Not enough for full job send:
image

Service settings:
image

Send while at the limit:
image
